modules/reference/pages/internal-metrics-reference.adoc

```diff
@@ -65,6 +65,46 @@ Can indicate latency caused by disk operations.
 
 ---
 
+=== vectorized_kafka_quotas_balancer_runs
+
+Number of times the throughput quota balancer has executed.
+
+*Type*: counter
+
+---
+
+=== vectorized_kafka_quotas_quota_effective
+
+Current effective quota, in bytes per second.
+
+*Type*: counter
+
+---
+
+=== vectorized_kafka_quotas_throttle_time
+
+Sequence of throttle times, in seconds.
+
+*Type*: histogram
+
+---
+
+=== vectorized_kafka_quotas_traffic_intake
+
+Total amount of Kafka traffic (in bytes) taken in from clients for processing.
+
+*Type*: counter
+
+---
+
+=== vectorized_kafka_quotas_traffic_egress
```

Comment:
@BenPope how are these different from the normal overall kafka throughput metrics?  If they are the doc description doesn't make it clear how.

Reply:
> @BenPope how are these different from the normal overall kafka throughput metrics? If they are the doc description doesn't make it clear how.

This is the amount of traffic that was considered by the throttler. So it depends on `kafka_throughput_controlled_api_keys`.